Xamarin.Forms.Platform.iOS/IVisualElementRenderer.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace Xamarin.Forms.Platform.iOS
 {
-	public interface IVisualElementRenderer : IDisposable, IRegisterable
+	public interface IVisualElementRenderer : IDisposable, IRegisterable, IEffectControlProvider
```

Comment:
Should any `VisualElement` have the ability to attach effects? I thought this would be okay. If not, this could be moved back.